### DISCUSSION TITLE
What is the highest point in Belarus?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#89] [Galaxy Trivia] [answered]
**Hint:** ⛰🤏

**Comment by Tynab** [comment, marked as answer]
The highest point in Belarus is Dzyarzhynskaya Hara (also known as Dzerzhinskaya Hill or Mount Dzerzhinsky), which reaches a height of 345 meters (1,132 feet) above sea level. It is located in the Minsk Region, about 40 kilometers (25 miles) northwest of the country's capital, Minsk. The hill is named after Felix Dzerzhinsky, a Soviet politician and founder of the Soviet secret police. While it is not particularly tall compared to the highest peaks in other countries, Dzyarzhynskaya Hara is an important landmark and symbol for the people of Belarus.
